### DISCUSSION TITLE
Hardware Acceleration Issues [Windows 11 iGPU]

### DISCUSSION THREAD

**Discussion by StrageFaithApernum2019** [ppy/osu#19765] [Q&A]
### Type

Performance

### Bug description

I was recently trying to get some plays during the time and I noticed that my performance in Osu Lazer's current update and yet somehow the motion was slightly delayed or my input was slow

Laptop which is normally good yet somehow it still was not working well

Just by chance I found out that there was one setting which is Hardware Acceleration

Normally at handling Hardware Acceleration when needed however out of curiousness I disabled Hardware Acceleration and the Motion is now similar to what the current Osu Stable feels like

I am wondering if this is only affecting Computers that may have iGPUs / Mobile CPUs or is this due to the computer not being able to handle Hardware Acceleration

[CPU: Ryzen 5500u]

### Version

2022.810.2

### Logs

[database.log](https://github.com/ppy/osu/files/9333712/database.log)
[performance.log](https://github.com/ppy/osu/files/9333713/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9333714/runtime.log)

**Comment by peppy** [comment]
probably nvidia optimus. please search for existing issues, there are many of these.

things will improve when we add direct x support.

**Reply by StrageFaithApernum2019** [reply]
There is no Nvidia Support when I have discovered the issues mentioned

However thank you for the information will look forward to it
